.github/pull_request_template.md

```diff
@@ -1,10 +1,13 @@
-<!--
-Thank you for sending the PR! We appreciate you spending the time to work on these changes.
-
-Please make sure that you added appropriate entry in docs/Changelog.md and docs/MigrationGuide.md
-
-You can learn more about contributing to Nussknacker here: https://github.com/TouK/nussknacker/blob/staging/CONTRIBUTING.md
-
-Happy contributing!
-
--->
+## Describe your changes
+
+## Checklist before merge
```

Comment:
WDYT about "adding jira number in PR title" point? If some jira exists, obviously.

Reply:
I added it. However, take a look if it is fine for you. I tried to add it in the form that will be suitable both for internal PRs and external contributions